src/flashbar/index.tsx

```diff
@@ -24,7 +24,7 @@ export default function Flashbar(props: FlashbarProps | CollapsibleFlashbarProps
   }
 }
 
-function isStackedFlashbar(props: any): props is CollapsibleFlashbarProps {
+function isStackedFlashbar(props: FlashbarProps) {
```

Comment:
is this needed anymore now that `stackItems` exists on the main interface?

Reply:
Just simplified this